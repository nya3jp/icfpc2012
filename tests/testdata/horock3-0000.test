RRRRRRRRRRRRRRRRLWRULULULULULULULRULLRDLUUURRRDDDDDDDDDDRULLLLDLLLUUULLUUUUURUUURRRDRRDRDRRRRDDDDRRRUUUUULUULLDRRDA
#################
# \\\\#   *  *  #
#  ##   !!#..@. ####
#*    @...####.....#
#\\   \@...........#
#\  # .\@    #  #.########
#\  # ..\@   #   ..      #
#\  # .. \@  #   ..      #
#\  # *.. \@    #######  #
#\  # ###  \@   #     #W #
### #       \@  #     ##L#
#...#@@**@@**\@ #
#R ..........!\##
#################

Growth 25
