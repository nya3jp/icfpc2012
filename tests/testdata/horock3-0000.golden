Step 114 - Score 1586 - ABORT_END
Water: 0 / 10, Level 0
Growth: 6 / 25, Razor: 3
#################         
#     #   **    #         
#  ##     #.    ####      
#         ####R ...#      
#            .. ...#      
#   #        #  #.########
#   # ..     #   .. WWWWW#
#   # ..     #   .. WWWWW#
#   # *..       #######WW#
#*  # ###       #     #WW#
### #           #     ##O#
#.. #           #         
#      **  **  ##         
#################         

